src/serializer/Emitter.js

```diff
@@ -504,4 +504,17 @@ export class Emitter {
     invariant(!this._finalized);
     return new BodyReference(this._body, this._body.entries.length);
   }
+  emitLazyReactElementInitializer(statement: BabelNodeStatement): void {
```

Comment:
I'd prefer it if we don't add such special cases to the Emitter. The Emitter is already confusing enough...